lib/utils/stripe-client.ts

```diff
@@ -0,0 +1,13 @@
+import { loadStripe, Stripe } from "@stripe/stripe-js";
+
+let stripePromise: Promise<Stripe | null>;
+
+export const getStripe = () => {
+  if (!stripePromise) {
+    stripePromise = loadStripe(
+      process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY_LIVE ?? process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY ?? ""
```

Comment:
Do we need a different key for the deploy preview and dev? Also, considering some devs might not have experience with Stripe, should we add context on how to test stripe somewhere? https://stripe.com/docs/testing

Reply:
deploy preview is generally set to `dev` variables but we can handle anything since the new netlify updates ❤️